OrbitCore/LinuxPerfData.h

```diff
@@ -17,13 +17,24 @@ class LinuxPerfData
 {
 public:
     std::string m_header = "";
-    uint32_t m_tid = 0;
+    uint32_t m_tid = 0; // TODO: This is actually redundant
```

Comment:
How so?  Could we remove it?

Reply:
It is also stored as m_ThreadId in the underlying callstack. I removed it.